com.unity.shadergraph/Documentation~/TableOfContents.md

```diff
@@ -31,7 +31,6 @@
     * [Property Types](Property-Types)
     * [Keywords](Keywords)
     * [Data Types](Data-Types)
-    * [Precision Types](Precision-Types)
```

Comment:
Did you mean to remove this?

Reply:
Yes. Discussed with Chris back during review, it replicated part of the Precision Modes doc and was confusing in its relationship with that.